types/function_list_builder.go

```diff
@@ -23,21 +24,64 @@ type FunctionLookupBuilder struct {
 	TopicDelimiter string
 }
 
-// Build compiles a map of topic names and functions that have
-// advertised to receive messages on said topic
-func (s *FunctionLookupBuilder) Build() (map[string][]string, error) {
-	var err error
+//getNamespaces get openfaas namespaces
+func (s *FunctionLookupBuilder) getNamespaces() ([]string, error) {
```

Comment:
Can we add unit test for this ? We can stub the JSON response.

Reply:
Added unit tests.